go/libraries/doltcore/sqle/dfunctions/active_branch.go

```diff
@@ -25,33 +25,39 @@ import (
 
 const ActiveBranchFuncName = "active_branch"
 
+// Deprecated: please use the version in the dprocedures package
 type ActiveBranchFunc struct {
 }
 
 // NewActiveBranchFunc creates a new ActiveBranchFunc expression.
+// Deprecated: please use the version in the dprocedures package
```

Comment:
No need to deprecate / convert to a procedure, this is strictly functional with no side effects